website/client/js/services/userServices.js

```diff
@@ -319,7 +319,9 @@ angular.module('habitrpg')
 
         addTag: function(data) {
           user.ops.addTag(data);
-          Tags.createTag(data.body);
+          Tags.createTag(data.body).then(function(res) {
+            sync();
```

Comment:
After looking into #7820 a bit more, I noticed the tag is being added just fine to the users tags, but the id seems to be incorrect until it syncs. Would it be better to wait for the response with the correct id before adding it to the users tags since it isn't usable before then anyways?